src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreWorker.cs

```diff
@@ -45,11 +46,11 @@ internal sealed class SolutionRestoreWorker : SolutionEventsListener, ISolutionR
         private readonly Lazy<IVsSolutionManager> _solutionManager;
         private readonly Lazy<INuGetLockService> _lockService;
         private readonly Lazy<Common.ILogger> _logger;
-        private readonly AsyncLazy<IComponentModel> _componentModel;
+        private readonly Microsoft.VisualStudio.Threading.AsyncLazy<IComponentModel> _componentModel;
```

Comment:
This change shouldn't be necessary. This comment applies to all occurrences where the fully qualified name was changed from simply referencing AsyncLazy.

Reply:
fixed in the latest commit